### ISSUE TITLE
Chirps data fetching not working

### ISSUE THREAD

**Issue by pahanc** [ropensci/chirps#27]
Hi, I am a mac user with RStudio version 4. I am finding that the data fetching script in the Readme is not able to fetch the data. I am running the following commands (directly from the Readme):

lonlat <- data.frame(lon = c(-55.0281,-54.9857, -55.0714),
                     lat = c(-2.8094, -2.8756, -3.5279))

dates <- c("2017-01-01", "2017-12-31")

dat <- get_chirps(lonlat, dates)

At this point, get_chirps times out. I have not been able to access the chirps ftp server either (ftp://ftp.chg.ucsb.edu/pub/org/chg/products/CHIRPS-2.0/global_daily/tifs/p05/)

**Comment by kauedesousa** [comment]
Hi @pahanc

I think this issue was due to some maintenance in the ClimateServ server, which was also reported some months ago in this issue https://github.com/ropensci/chirps/issues/26#issue-807743172

I just tested and it is working fine. Can you confirm that it is working from your side? 

When this issue happens with the function, I try to check if the server is working with this path (copy and paste on my web browser)

```
https://climateserv.servirglobal.net/chirps/getDataFromRequest/?id=46f3a596-e9bc-4448-9a7b-e37ff15631c2
```

If it returns this it means that the server is working fine. If not, it means that there is an issue in the R package.

```
{"data": [{"date": "4/1/2018", "workid": "4fc3b5e2-fc9e-4109-83f1-6a90fc0e6b61", "epochTime": "1522540800", "value": {"avg": 8.716950627363355}}, {"date": "4/2/2018", "workid": "af9341ac-ec5e-4e56-b469-bada1f0bfe4d", "epochTime": "1522627200", "value": {"avg": 12.231267725163287}}, {"date": "4/3/2018", "workid": "2b4982b6-7e25-4ec8-af8f-23f0efc68c8f", "epochTime": "1522713600", "value": {"avg": 6.144928723358542}}, {"date": "4/4/2018", "workid": "781c1ff3-a0d2-4776-a24b-af0b0f40ee9f", "epochTime": "1522800000", "value": {"avg": 5.046160622206944}}, {"date": "4/5/2018", "workid": "eb8ffd1f-b0fa-4799-9efa-2830cd9102c8", "epochTime": "1522886400", "value": {"avg": 4.728046982425232}}, {"date": "4/6/2018", "workid": "a0c05e56-2ec6-40a6-ab55-f1b5f178c40c", "epochTime": "1522972800", "value": {"avg": 8.19946717084909}}, {"date": "4/7/2018", "workid": "9a58dde0-42d7-4a06-9237-881cd1c4f075", "epochTime": "1523059200", "value": {"avg": 4.953129364150052}}, {"date": "4/8/2018", "workid": "573994f0-0142-40ec-b1c6-25e91810a488", "epochTime": "1523145600", "value": {"avg": 8.139506434771398}}, {"date": "4/9/2018", "workid": "d72d8354-2b41-43c5-b0b4-c362f055c15d", "epochTime": "1523232000", "value": {"avg": 8.067022602268821}}, {"date": "4/10/2018", "workid": "b979c0e0-68e7-47bc-bf54-410901299435", "epochTime": "1523318400", "value": {"avg": 6.447803422567893}}, {"date": "4/11/2018", "workid": "dadfd6c3-91a4-4741-aba0-c427cab2e8f1", "epochTime": "1523404800", "value": {"avg": 15.417572619456857}}, {"date": "4/12/2018", "workid": "9c2ed372-fb8d-4cfb-91b2-a91e6902de48", "epochTime": "1523491200", "value": {"avg": 12.525282528360261}}, {"date": "4/13/2018", "workid": "72bf2e8d-9ead-416a-b396-b19319ea4df7", "epochTime": "1523577600", "value": {"avg": 9.972868414833275}}, {"date": "4/14/2018", "workid": "d815ab51-d665-4ca2-9849-897c4b4a3df3", "epochTime": "1523664000", "value": {"avg": 6.685189014695772}}, {"date": "4/15/2018", "workid": "9d7446b8-ab18-47c1-ac65-5b2a90fdeeeb", "epochTime": "1523750400", "value": {"avg": 1.3178828433847543}}, {"date": "4/16/2018", "workid": "8d072707-8a6c-4f0d-b5bb-21dde7562a55", "epochTime": "1523836800", "value": {"avg": 0.785841169780208}}, {"date": "4/17/2018", "workid": "ac6ba3cd-536a-418a-a20a-af8472e4a676", "epochTime": "1523923200", "value": {"avg": 8.278863145625644}}, {"date": "4/18/2018", "workid": "eaf1d8af-daf1-4382-8e38-22ad68197809", "epochTime": "1524009600", "value": {"avg": 0.07900738183392328}}, {"date": "4/19/2018", "workid": "dd254cb0-6b43-4846-a806-db2c0afb03a2", "epochTime": "1524096000", "value": {"avg": 0.8416194555152114}}, {"date": "4/20/2018", "workid": "ac51b6fc-6dd6-44dc-bac1-79557db9cd15", "epochTime": "1524182400", "value": {"avg": 2.75319472568967}}, {"date": "4/21/2018", "workid": "b677cb18-1598-414e-807e-f3681274622b", "epochTime": "1524268800", "value": {"avg": 4.616690658301822}}, {"date": "4/22/2018", "workid": "c11f5ad8-5cb6-46ce-8b64-49b3efee53e6", "epochTime": "1524355200", "value": {"avg": 9.151235255886903}}, {"date": "4/23/2018", "workid": "32af0cc2-39f8-4a71-904d-3f3ed3ab40a0", "epochTime": "1524441600", "value": {"avg": 0.12194402490788286}}, {"date": "4/24/2018", "workid": "21111909-a8d6-4a1d-9e97-a113ca39bb17", "epochTime": "1524528000", "value": {"avg": 2.9939999650867994}}, {"date": "4/25/2018", "workid": "5e41f406-5296-4ec0-bca4-1ae4f4eca4f9", "epochTime": "1524614400", "value": {"avg": 1.2167549120187349}}, {"date": "4/26/2018", "workid": "69155711-dfd7-458a-b22d-f4aae2071385", "epochTime": "1524700800", "value": {"avg": 0.6270308824042842}}, {"date": "4/27/2018", "workid": "88016116-177c-4893-8d46-a6c4329482cb", "epochTime": "1524787200", "value": {"avg": 1.7732209710166724}}, {"date": "4/28/2018", "workid": "2bbe7295-c1fb-4480-a539-f3bbe4fd97b5", "epochTime": "1524873600", "value": {"avg": 16.41208050446889}}, {"date": "4/29/2018", "workid": "12a45a9a-0a68-49a3-91c1-1c2ac19010c2", "epochTime": "1524960000", "value": {"avg": 6.5620763417411485}}, {"date": "4/30/2018", "workid": "3378b886-fa92-4987-bc3b-7ea33a4fea4f", "epochTime": "1525046400", "value": {"avg": 16.857927444998282}}]}
```

Best regards

**Comment by pahanc** [comment]
Hi Kaue de Sousa, thanks for your reply. I have just checked it again and
it works fine now.
All the best,
Penny

**Comment by kauedesousa** [comment]
Now you can use the new solution available from chirps (>= 0.1.3) as I show here https://github.com/ropensci/chirps/issues/28#issuecomment-866817419

**Comment by EqUaL90** [comment]
Hi. Currently, climateserv seems to be down. Has it been decommissioned? 
I can't access the website.
